packages/harvester/harvester-csi-driver/generated-changes/patch/Chart.yaml.patch

```diff
@@ -1,20 +1,14 @@
 --- charts-original/Chart.yaml
 +++ charts/Chart.yaml
-@@ -1,13 +1,14 @@
- annotations:
-   catalog.cattle.io/certified: rancher
-   catalog.cattle.io/display-name: Harvester CSI Driver
--  catalog.cattle.io/kube-version: '>= 1.18.0-0 < 1.24.0-0'
-+  catalog.cattle.io/kube-version: '>= 1.18.0-0 < 1.25.0-0'
+@@ -5,9 +5,10 @@
    catalog.cattle.io/namespace: kube-system
    catalog.cattle.io/os: linux
--  catalog.cattle.io/permits-os: linux
-+  catalog.cattle.io/permits-os: linux,windows
--  catalog.cattle.io/rancher-version: '>= 2.6.1-0 < 2.7.0-0'
+   catalog.cattle.io/permits-os: linux
```

Comment:
`windows` was added to the label previously, but now is dropped. Is this an intended change? 
So the chart can not be installed in Windows anymore? 
(same for the chart harvester-cloud-driver)

Reply:
Hi @jiaqiluo, follow the above comment https://github.com/rancher/charts/pull/2450#discussion_r1133391434, so I remove `windows` from the permits-os.

Hi @guangbochen, do we need to support windows now? Thanks!

Reply:
Yep, Windows node support is not in our scope yet, so we should be fine to remove it, thanks.